stuntidp-originauth-key-priv-json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
This doesn't look very sensible. Is there a good reason for it being committed?

Reply:
none whatsoever